translation/translator.go

```diff
@@ -54,10 +62,10 @@ func (t *Translator) Choice(key string, number int, options ...translationcontra
 		locale = options[0].Locale
 	}
 
-	return makeReplacements(t.selector.Choose(line, number, locale), replace), nil
+	return makeReplacements(t.selector.Choose(line, number, locale), replace)
 }
 
-func (t *Translator) Get(key string, options ...translationcontract.Option) (string, error) {
+func (t *Translator) Get(key string, options ...translationcontract.Option) string {
```

Comment:
There is a problem, the files: 

`lang/cn.json`:

```
{
  "hello": "cn-world"
}
```

`json/en.json`, 

```
{
  "hello": "en-world",
  "hello1": "en-world1"
}
```

The configuration:

```
"locale": "cn",
"fallback_locale": "en",
```

`facades.Lang(context.Background()).Get("hello1")` will be returned `hello1`, but expect `en-world1`.

So the logic of this method should be:

1. Get value from `lang/cn.json`;
2. If `lang/cn.json` exists, but the key doesn't exist, get value from `lang/en.json`, if `lang/en.json` doesn't exist, get value from `lang/en/*`;
3. If `lang/cn.json` doesn't exist, get value from `lang/cn/*`;
4. If `lang/cn/*` doesn't exist, get value from `lang/en.json`, if `lang/en.json` doesn't exist, get value from `lang/en/*`;

Reply:
Fallback translations are not allowed for `lang/en.json`

Reply:
Fallback translations are not allowed for `lang/cn.json`(It is intended since laravel doesn't support it)